### PR TITLE
Website fixes from warsztatywww.pl

### PR DESCRIPTION
**Pull request by cytadela8**
- Added "comment" to Paricipants view
- Fixed facebook button (to comply with facebook policy)
- Removed Wirchomla fields from profile
- 150 -> 50 words in letter (text change)

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -1,20 +1,22 @@
 {% extends "base.html" %}
 
 {% block content %}
-<div class="container">
+<div class="container" style="max-width: 400px">
```
Comment (radrow) [inline templates/login.html]: What, move to CSS

File: templates/login.html [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
...vHUWS6qVMp6Ym4MyUU8YGGxmMJAGtDPYoKLmD9h8oxWFoaP0M/jIzzR3CACXA4cO0YWFaLrENF+KAN36soxmDBvCikxXSFv6nxAGits6GouArfr8lt1YD49OnJmDokCHHBPo2zrh2A1WFDzYUG5BdWYPcW99vQFUdMPP4wIxJJURYE+TVtXwdq6myoLL+I7ANtPx90NG05j+uBgcyFtnwGb20HIiIiap22nMf1wC5LIiIiIp0xkBERERHpjIGMiIiISGcMZEREREQ6YyAjIiIi0hkDGREREZHOGMi8rOayB6uPEhERETnpkIFs2/5zWkU6rO3ZnfezExERkT46ZCB76Z39nTKUfbPjNP6x9rBWMSIiIiKJDrlSPxEREVFbcKV+IiIiImoRBjIiIiIinTGQEREREemMgYyIiIhIZwxkRERERDpjICMiIiLSGQMZERERkc4YyIiIiIh05tVAZjAIWkWIiIiI2lVnzCNeDWThwV21ihARERG1q86YR7wayAb1DdYqQkRERNSuOmMe8Wog+3FapFYRIiIionbVGfOIVwPZrSnhSE4I0SpGRERE1C6SE0Jwa0q4VrEOx6uBDAAW3JPYKQfTERERUedmMAhYcE+iVrEOyeuBrF9cIJ57+BaGMiIiIrpuDAYBzz18C/rFBWoV7ZAEURRFrUKtkVd4ESs+zkVOfoVWUSIiIqJWS04IwYJ7EjttGAPaMZA12b7/HP6bVYLDxytxrvIyHI52/XVERER0gzMYBIQHd8WgvsH4cVpkpxwzJtfugYyIiIiI3PP6GDIiIiIiahkGMiIiIiKdMZARERER6YyBjIiIiEhnDGREREREOmMgIyIiItIZAxkRERGRzhjIiIiIiHTGQEZERESkMwYyIiIiIp0xkBERERHpjIGMiIiISGcMZEREREQ6YyAjIiIi0hkDGREREZHOGMiIiIiIdMZARkRERKQzBjIiIiIinTGQEREREemMgYyIiIhIZwxkRERERDpjICMiIiLSGQMZERERkc4YyIiIiIh0xkBGREREpDMGMiIiIiKdMZARERER6YyBjIiIiEhnDGREREREOmMgIyIiItIZAxkRERGRzhjIiIiIiHTGQEZERESkMwYyIiIiIp0xkBERERHpjIGMiIiISGcMZEREREQ6YyAjIiIi0tn/B17o/REI8gDnAAAAAElFTkSuQmCC">
```
Comment (radrow) [inline templates/login.html]: What the fuck

File: templates/login.html [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+		</a>
+                <a class="button zocial google" href="{% url 'scopedallaccess-login' 'google' %}" style="max-width: 220px">
```
Comment (radrow) [inline templates/login.html]: Remove style

**Comment by cytadela8** [comment]
OK!

**Pull request merged**